src/redux/books/books.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const bookSlice = createSlice({
+  name: 'book',
+  initialState: {
+    value: [],
+  },
+  reducers: {
+    addBook: (state, action) => {
+      state.value.push(action.payload);
+    },
+
+    removeBook: (state, action) => {
+      state.value = state.value.filter((book) => book.id !== action.payload.id);
+    },
```

Comment:
- [ ] Please fix your addBook and removeBook method is not working.

Reply:
Hi @CollinsTatang thank you for your feedback.
As for the functionality I understood that for the first submit I only needed to setup the reducers, and as for the reducers I tested them and they do work when I called them on another component and did the necessary configurations, but that is the next requirement for the project

Reply:
Hi @fasaldana 
The project required the addBook function and the removeBook function to be working properly.

Reply:
This is not yet implemented.

Please fix your addBook and remove book method is not working.

User can not Add a book nor Remove a book.

Reply:
Hi @Emmyn5600 this is day 1 project of the week, and the requirements says only configuration not functionality, but if there is really something I'm missing or I misread in the requirements then I will do the functionality, thanks for the feedback.

Reply:
@Emmyn5600  but then I don't know what I should submit for day 2 where they're asking for the functionality 😅

Reply:
@Emmyn5600 Here is a link of the requirements for this project
[https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md](url)

Reply:
Hi @fasaldana 
Sorry I miss it let me signal for correction

Reply:
Thank you @CollinsTatang, I appreciate it.